services/web/server/tests/unit/with_dbs/09/test_projects_01.py

```diff
@@ -960,3 +1011,60 @@ async def test_delete_multiple_opened_project_forbidden(
         if user_role != UserRole.ANONYMOUS:
             pytest.fail("socket io connection should not fail")
     await _delete_project(client, user_project, expected_forbidden)
+
+
+# PAGINATION
+def standard_user_role() -> Tuple[str, Tuple[UserRole, ExpectedResponse]]:
+    all_roles = standard_role_response()
+
+    return (all_roles[0], [pytest.param(*all_roles[1][2], id="standard user role")])
+
+
+@pytest.mark.parametrize("limit", [7, 20, 43])
```

Comment:
So if limits are e.g. negative, the pydantic model wuld raise ValidationError ... did you check that this is correctly translated into an ``web.HTTPBadRequest`` or ``HTTPUnprocessableEntity`` error? Something like [this](https://github.com/ITISFoundation/osparc-simcore/blob/18f60423183216149c75f310304ffff479aae1ee/services/web/server/src/simcore_service_webserver/catalog_api_handlers.py#L84) but I guess we could do better :-)

Reply:
yes I forgot that test... was actually in the pipeline. thanks for reminding me.